### PR TITLE
Added TC-179

### PR DESCRIPTION
**Pull request by isenkovska**

### CODE REVIEW

**Review by o-akhadova — CHANGES_REQUESTED** [review]

File: test/extended/story.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    });
+    beforeEach(() => {
+        browser.refresh();
+    });
+
+    it('TC-179 The word \'year\' appears in a story after filling in the field age \'1\'. ', function () {
+        inputValues4(name.default, gender.it, age.one, story.comedy);
+        $(sel.submit).click();
+        let story1 = $$(sel.story)[0].getText();
+        expect (story1).toContain(exp.year_singular);
```
[the remark this answers is not in the document]
Comment (isenkovska) [reply test/extended/story.js]: thanks, everything should work in the recent commit, please check it out

[the remark this answers is not in the document]
Comment (o-akhadova) [reply test/extended/story.js]: Good job, everything works! 
But you should change summary for TC-180 to "TC-180 The word 'years' appears in a story after filling in the field age '10'. "

[the remark this answers is not in the document]
Comment (ivansh83) [reply test/extended/story.js]: Yeah, I agree.

[the remark this answers is not in the document]
Comment (isenkovska) [reply test/extended/story.js]: thanks, fixed the summary for TC-180

**Review by irinadash — APPROVED** [review]

File: test/extended/story.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (isenkovska) [reply test/extended/story.js]: everything should be good now

**Review by o-akhadova — APPROVED** [review]
Good job!

**Pull request merged**